src/NuGet.ProjectModel/LockFile.cs

```diff
@@ -61,5 +62,16 @@ public bool IsValidForPackageSpec(PackageSpec spec)
 
             return true;
         }
+
+        // DNU REFACTORING TODO: temp hack to make generated lockfile work with runtime lockfile validation
+        private static string RuntimeStyleLibraryRangeToString(LibraryRange libraryRange)
```

Comment:
We need to change implementation of `LibraryRange.ToString()` in runtime but I want to make the PRs for refactoring minimal. That's why I introduce this hack.